.github/workflows/ruby.yml

```diff
@@ -24,13 +24,13 @@ jobs:
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: 3.3
 
     - name: Install bundler
-      run: gem install bundler -v 2.1.1
+      run: gem install bundler
 
     - name: Install dependencies
-      run: bundle _2.1.1_ install
+      run: bundle install
 
     - name: Run tests
       run: bundle exec rake spec
```

Comment:
issue: Tests don't run with this task in Ruby 3. A simple way to fix this is to replace it with:

```
bundle exec rspec
```

Reply:
Nice catch, I've updated & pushed it up.